unittests/tools/test_api_msdefender_parser.py

```diff
@@ -0,0 +1,43 @@
+from ..dojo_test_case import DojoTestCase
+from dojo.tools.api_msdefender.parser import ApiMSDefenderParser
+from dojo.models import Test
+
+
+class TestAPIMSDefenderAPIParser(DojoTestCase):
+
+    def test_parse_many_findings(self):
+        filelist = []
+        vulnerabilities = open("unittests/scans/api_msdefender/report_many_vulns.json")
+        filelist.append(vulnerabilities)
+        machines = open("unittests/scans/api_msdefender/machines.json")
+        filelist.append(machines)
+        parser = ApiMSDefenderParser()
+        findings = parser.get_findings(filelist, Test())
+        self.assertEqual(4, len(findings))
+        finding = findings[2]
+        self.assertEqual("Medium", finding.severity)
+        self.assertEqual("wjeriowerjoiewrjoweirjeowij-_-CVE-5678-9887-_-packagvendor-_-tools-_-1.2.3.4-_-", finding.title)
+
```

Comment:
Can you add endpoint validation, please? https://documentation.defectdojo.com/contributing/how-to-write-a-parser/#endpoint-validation

Reply:
And also test endpoint, please